### DISCUSSION TITLE
My k3s cluster powering down with data anomalies

### DISCUSSION THREAD

**Discussion by MagicStarTrace** [k3s-io/k3s#8713] [Q&A]
The cluster hasn't been backed up yet, and I see the pod data is gone. Is there any way to get the cluster working again? Upgrade or reinstall or something?

**Comment by MagicStarTrace** [comment]
![image](https://github.com/k3s-io/k3s/assets/15685376/548cc0a7-00dc-4e53-a982-28e8ba00da3a)
I see that none of the processes related to kube-rbac-proxy on the master node exist anymore

**Comment by brandond** [comment]
I have no idea what you mean by "powering down" and I have no idea what pod you're talking about. It sounds like you experienced some data loss when your K3s node was unexpectedly shut down, and are trying to figure out if you can get some data out of a pod that was running before the host restarted?

Unless you had your data stored on a persistent volume, then no, it is likely gone. Tempfs volumes and writes to image filesystem are discarded when the pod terminates.

**Reply by MagicStarTrace** [reply]
The esxi server is down unexpectedly due to a power outage.
It looks like the vm virtual machine that corresponds to k3s lost its data.

Hard drive failure or SQLite data loss?
under namespace:
No resources were found in the service namespace.
